name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/KeytradeBankPDFExtractor.java

```diff
@@ -64,17 +64,19 @@ private void addBuySellTransaction()
                 // Achat 310 HOME24 SE  INH O.N. (DE000A14KEB5) à 15,9324 EUR
                 // Vente 310 VERBIO VER.BIOENERGIE ON (DE000A0JL9W6) à 33,95 EUR
                 // Verkauf 22 SARTORIUS AG O.N. (DE0007165607) für 247 EUR
-                .section("isin", "name", "shares", "currency")
-                .match("^(Kauf|Achat|Verkauf|Vente) (?<shares>[\\.,\\d]+) (?<name>.*) \\((?<isin>[\\w]{12})\\) .* (?<currency>[\\w]{3})$")
+                .section("isin", "name", "shares", "amount", "currency")
```

Comment:
Including the amount here is also wrong.
We generate by this part only the security in the appropriate currency, if this does not exist yet. 
The amount would then be the market value and would be overwritten further down in the [part](https://github.com/buchen/portfolio/blob/53a59729777988c9e388dd08c274aac96b62db42/name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/KeytradeBankPDFExtractor.java#L84).

To understand how the security is created or created in the corresponding currency in PP, you can take a look [here](https://github.com/buchen/portfolio/blob/53a59729777988c9e388dd08c274aac96b62db42/name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/AbstractPDFExtractor.java#L173).